go.mod

```diff
@@ -105,7 +105,7 @@ require (
 	github.com/mdlayher/netlink v1.7.2
 	github.com/microsoft/go-mssqldb v0.0.0-00010101000000-000000000000 // replaced
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/moby/term v0.0.0-20221128092401-c43b287e0e0f
+	github.com/moby/term v0.5.0
```

Comment:
https://github.com/moby/term/compare/c43b287e0e0f...v0.5.0

Seems alright from commit title, but I'm always wary of 0.x jumps.